src/styles/layouts/_layout-account.scss

```
@@ -0,0 +1,65 @@
+.layout-account {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: rem(40px) repeat(2, 1fr) rem(200px);
+  grid-template-areas:
+    'header'
+    'sidebar'
+    'main'
+    'footer';
+
+  height: 100vh;
+
+  @include respond-above('md') {
+    grid-template-rows: rem(80px) repeat(2, 1fr) rem(250px);
+  }
+
+  @include respond-above('lg') {
+    grid-template-columns: minmax(350px, 450px) minmax(642px, 1fr);
+    grid-template-rows: rem(100px) 1fr rem(300px);
```

Would be better to add `column-gap` here to split `sidebar` and `main` apart instead of padding.
```suggestion
    grid-template-rows: rem(100px) 1fr rem(300px);
    column-gap: some_value;
```